lighthouse/helpers/plates.py

```diff
@@ -215,7 +215,7 @@ def query_for_cherrypicked_samples(rows: Optional[List[SampleDoc]]) -> Optional[
                 FIELD_ROOT_SAMPLE_ID: getattr(row, FIELD_DART_ROOT_SAMPLE_ID),
                 FIELD_RNA_ID: getattr(row, FIELD_DART_RNA_ID),
                 FIELD_LAB_ID: getattr(row, FIELD_DART_LAB_ID),
-                FIELD_RESULT: {"$regex": "^positive", "$options": "i"},
+
```

Comment:
So, this method takes some samples from DART, and queries MongoDB to find the matching ones.
@emrojo 's suggestion on the issue for this was to either:
- match on the 4 fields, root sample id, rna id, lab id and result, but instead of hardcoding the result to 'positive', compare the value in DART with the value in Mongo
- OR, match based on uuid

I see there's a problem with doing the first one, which is that DART doesn't hold the 'result' information (or, if it does, it's not in the view that we query from).

I think that matching only on root sample id, rna id and lab id, as you've got here, is inadequate, because the MongoDB unique key includes all 4 fields, so you might have more than one row with the same root sample id, rna id and lab id. 

Therefore, I think we should probably switch it to use @emrojo 's second suggestion - the uuid. It looks to me like it's called 'lh_sample_uuid' in both databases.